environments/common.yaml.gotmpl

```diff
@@ -64,114 +97,12 @@ vault:
   envs:
     - VAULT_ADDR: "http://localhost:8200"
 
-kafka:
-  enabled: true
-  namespace: {{ env "KAFKA_NAMESPACE" | default "kafka" }}
-  auth:
-    enabled: true
-    username: user1
-    password: secret
-  externalAccess:
-    enabled: {{ env "KAFKA_EXTERNAL_ACCESS" | default false }}
-
-zookeeper:
-  auth:
-    enabled: true
-    serverUsers: zookeeperUser
-    serverPasswords: zookeeperPassword
-    clientUser: zookeeperClientUser
-    clientPassword: zookeeperClientPassword
-
-redis:
-  enabled: true
-  namespace: {{ env "REDIS_NAMESPACE" | default ( env "ORCHESTRATE_NAMESPACE" | default $defaultNamespace ) }}
-  password: such-secret
-
-postgresql:
-  enabled: true
-  namespace: {{ env "POSTGRES_NAMESPACE" | default ( env "ORCHESTRATE_NAMESPACE" | default $defaultNamespace ) }}
-  host: {{ env "POSTGRES_HOST" | default "postgresql" }}
-  port: 5432
-  username: {{ env "POSTGRES_USER" | default "api"}}
-  password: {{ env "POSTGRES_PWD" | default "such-secret" }}
-  database: {{ env "POSTGRES_DB" | default "api" }}
-
-postgresqlHA:
-  enabled: false
-  namespace: {{ env "POSTGRES_NAMESPACE" | default ( env "ORCHESTRATE_NAMESPACE" | default $defaultNamespace ) }}
-  postgresql:
-    username: api
-    password: such-secret
-    database: api
-    repmgrPassword: such-secret
-
 qkm:
-  enabled: true
-  url: "http://quorum-key-manager-quorumkeymanager:8080"
-  proto: https
-  fullname: "quorumkeymanager"
   orchestrate:
     storeName: "eth-accounts"
-    apiKey: {{ env "QKM_API_KEY" | default "NmwyMCB3YXMgaGVyZQ==" }}
+    apiKey: {{ env "QKM_API_KEY" | default "YWRtaW4tdXNlcg==" }}
```

Comment:
I would rather not have any default value

Reply:
This is the default QKM expected value, so it gives the user a good hint according to me. For that reason I would rather keep it